flink-runtime/src/main/java/org/apache/flink/runtime/checkpoint/channel/ChannelStateCheckpointWriter.java

```diff
@@ -108,32 +108,30 @@
 		runWithChecks(() -> serializer.writeHeader(dataStream));
 	}
 
-	void writeInput(InputChannelInfo info, Buffer... flinkBuffers) throws Exception {
+	void writeInput(InputChannelInfo info, Buffer flinkBuffers) throws Exception {
```

Comment:
Should be `flinkBuffer` (or simply `buffer`).

Reply:
Renamed to `buffer`.
Thanks.